sdk/include/platform/sunburst/platform-rgbctrl.hh

```diff
@@ -0,0 +1,89 @@
+#pragma once
+#include <cdefs.h>
+#include <stdint.h>
+
+/**
+ * An enum representing each of the Sonata's RGB LEDs.
+ */
+enum class SonataRgbLed
+{
+	Led0 = 0,
+	Led1 = 1,
+};
+
+/**
+ * A driver for the Sonata's RGB LED Controller
+ */
+struct SonataRgbLedController
+{
+	/**
+	 * Registers for setting the 8-bit red, green, and blue values
+	 * for the two RGB Leds.
+	 */
+	uint32_t ledColors[2];
+
+	/// Control Register Fields
+	enum [[clang::flag_enum]] : uint32_t{
+	  /// Write 1 to set RGB LEDs to specified colours.
+	  ControlSet = 1 << 0,
+	  /**
+	   * Write 1 to turn off RGB LEDs.
+	   * Write to ControlSet to turn on again.
+	   */
+	  ControlOff = 1 << 1,
+	};
+	/// Control Register (fields documented above)
```

Comment:
'Above' isn't great because the 'above' bit isn't visible in the mouse-over pop-up in IDEs (which is the main thing that we write doc comments for).  If you point at another enum by name, people can refer to that.